driver_cpl/cime_config/config_component.xml

```diff
@@ -2974,14 +2974,6 @@
     <desc>Whether to clean the test after it is built/run</desc>
   </entry>
 
-  <entry id="CCSM_BASELINE">
```

Comment:
Why has CCSM_BASELINE been removed for config_component.xml in the driver. How does a user change this via changing an xml file? Or is the idea that this will only be set on the commandline for create_test?